doc/classes/PhysicsServer3D.xml

```diff
@@ -1543,7 +1551,9 @@
 		<constant name="SHAPE_HEIGHTMAP" value="8" enum="ShapeType">
 			The [Shape3D] is a [HeightMapShape3D].
 		</constant>
-		<constant name="SHAPE_CUSTOM" value="9" enum="ShapeType">
+		<constant name="SHAPE_SOFT_BODY" value="9" enum="ShapeType">
+		</constant>
+		<constant name="SHAPE_CUSTOM" value="10" enum="ShapeType">
```

Comment:
Don't include unrelated changes caused by doctool in the commit. (Use `git add -p` and `git reset -p` to make partial commits.)

The same applies to other parts modified in this file.